packages/TorneloScoresheet/src/components/ChessSquare/ChessSquare.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { View } from 'react-native';
+import { colours } from '../../style/colour';
+import {
+  BoardPosition,
+  boardPositionToIdx,
+  Position,
+} from '../../types/ChessMove';
+import PieceAsset from '../PieceAsset/PieceAsset';
+import { styles } from './style';
+
+export type ChessSquareProps = {
+  square: BoardPosition;
+};
+
+const squareColor = (position: Position) => {
+  const [row, col] = boardPositionToIdx(position);
+
+  if (row % 2 === 0) {
+    if (col % 2 == 0) {
+      // even, even
+      return {
+        backgroundColor: colours.darkBlue,
+      };
+    }
+
+    //even, odd
+    return {
+      backgroundColor: colours.lightBlue,
+    };
+  } else {
+    if (col % 2 == 0) {
+      // odd, even
+      return {
+        backgroundColor: colours.lightBlue,
+      };
+    }
+
+    // odd, odd
+    return {
+      backgroundColor: colours.darkBlue,
+    };
+  }
+};
+
```

Comment:
This function can be simplified.

| col index | row index | sum of indexes | colour |
| --------- | ---------- | ---- | ------ |
| even | even | even | darkBlue |
| even | odd | odd | lightBlue |
| odd | even | even | darkBlue |
| odd | odd | odd | lightBlue |

Colour depends on `(row + col) % 2`

```TypeScript
const squareColor = (position: Position) => {
  const [row, col] = boardPositionToIdx(position);
  const isAlternateTile = (row + col) % 2 === 0
  return { backgroundColor: isAlternateTile ? colours.lightBlue : colours.darkBlue }
}
```